Makefile

```diff
@@ -37,31 +39,60 @@ SLJIT_LIR_FILES = $(SRCDIR)/sljitLir.c $(SRCDIR)/sljitExecAllocator.c $(SRCDIR)/
 	$(SRCDIR)/sljitNativeTILEGX_64.c \
 	$(SRCDIR)/sljitNativeX86_common.c $(SRCDIR)/sljitNativeX86_32.c $(SRCDIR)/sljitNativeX86_64.c
 
-all: $(BINDIR) $(TARGET)
+.PHONY: all examples
```

Comment:
I would not create examples by default.

Reply:
`all` is still the default make rule and doesn't build `examples` by default.

The phony rule just tells make that these targets aren't file-based targets (i.e. the target doesn't create a file named `all`). Clean should also be added as a phony target.